schematic/manifest/generator.py

```diff
@@ -1641,7 +1625,7 @@ def create_manifests(
         title: Optional[str] = None,
         strict: Optional[bool] = True,
         use_annotations: Optional[bool] = False,
-    ) -> Union[List[str], List[pd.DataFrame], BinaryIO]:
+    ) -> Union[List[str], List[pd.DataFrame]]:
```

Comment:
If you pick excel as output type, do you still get a list[str] or just a str?

Reply:
the current behavior is that you would still get a list of excel file paths but only the first one would get returned as a file to the user (which is now being handled on the API side.). Now that I think about it more, to have the endpoint works more efficiently, probably it would make more sense to only return one excel file path here..but this is something that we could address in a different PR if needed.